CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ CHANGELOG
 - Consider Compute fleet clean-up completed during cluster deletion when instances are either in shutting-down or terminated state.
   This is to avoid cluster deletion failure for instance types with longer termination cycles.
 - Allow cloudwatch dashboard to be enabled and alarms to be disabled in the `Monitoring` section of the cluster config.
+- Renaming pcluster.sh profile to cookbook_environment.sh profile which will be used only for cfn commands.
```

Comment:
Here we should explain the change from the user perspective. What's the consequence of this change for the user?